webapp/channels/src/components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -115,6 +115,16 @@ export default class QuickSwitchModal extends React.PureComponent<Props, State>
         }
     };
 
+    private onHide = () => {
+        this.props.onExited?.();
+        setTimeout(() => {
+            const textbox = document.querySelector('.SidebarChannelNavigator_jumpToButton') as HTMLElement;
+            if (textbox) {
+                textbox.focus();
+            }
```

Comment:
Could you rename `textbox` to `button` or `element` here? The current name is incorrect since it's copied from above